packages/components/src/styles/skins/_alpheios.scss

```diff
@@ -283,8 +283,11 @@ html {
   --alpheios-notification-important-bg: rgba(198, 73, 6, 0.31);
   --alpheios-notification-important-border-color: var(--alpheios-important-notification-color);
 
-  //Hint
+  --alpheios-notification-imporant-alt-color: #31595E;
+  --alpheios-notification-imporant-alt-bg: #B0C38D;
+  --alpheios-notification-imporant-alt-border-color: #31595E;
```

Comment:
just a typo in the style name - should be 'important' not 'imporant'

Reply:
Oh, that's a good catch, will rename